src/main/java/stepdefs/LoginStepDefs.java

```diff
@@ -0,0 +1,33 @@
+package stepdefs;
+
+import businessobjects.User;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.testng.Assert;
+import pageobject.HomePage;
+import pageobject.LoginPage;
+
+
+public class LoginStepDefs {
+
+    @Given("^Login page is opened$")
+    public void openLoginPage() {
+        LoginPage loginPage = new LoginPage();
+        loginPage.openURL();
+        Assert.assertTrue(loginPage.checkUsernameIsDisplayed(), "Username field is not displayed");
+        Assert.assertTrue(loginPage.checkPasswordIsDisplayed(), "Password field is not displayed");
+        Assert.assertTrue(loginPage.checkLoginButtonIsDisplayed(), "Login button is not displayed");
+    }
+
+    @When("^Populate ([^\"]*) and ([^\"]*) fields and login$")
```

Comment:
When test executed u will have in log
Populate yauhen_valodzin@epam.com.qa and Ceakt_1234 fields and login
As u can see it is a bit confusing. Better add field names at least